Task 3/script.js

```diff
@@ -12,3 +12,35 @@ būti stilizuota su CSS ir būti responsive;
 -------------------------------------------------------------------------- */
 
 const ENDPOINT = 'https://api.github.com/users';
+const showUsersBtn = document.getElementById('btn');
+const output = document.getElementById('output');
+const infoMsg = document.getElementById('message');
+
+const getUsers = async () => {
+  try {
+    const response = await fetch(ENDPOINT);
+    const data = await response.json();
+    renderCards(data);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const renderCards = (arr) => {
```

Comment:
Bendrai be priekaištų, tik prieš kiekvieną render paleidimą vertėtų išvalyti ankstesnį turinį, nes dabar kiek bespaustum show users, tiek vis prideda tų pačių elementų į pabaigą.